tools/bin/build_image.sh

```diff
@@ -42,7 +42,7 @@ if [ "$FOLLOW_SYMLINKS" == "true" ]; then
   # to use as the build context
   tar cL "${exclusions[@]}" . | docker build - "${args[@]}"
 else
-  JDK_VERSION="${JDK_VERSION:-17.0.1}"
+  JDK_VERSION="${JDK_VERSION:-17.0.4}"
```

Comment:
Curious what prompted the version bump here

Reply:
The latest [amazoncorretto:17.0.1](https://hub.docker.com/_/amazoncorretto/tags?page=1&name=17.0.1) is 9 months old already and can have some outdated packages.